components/base/wheeled/wheeled_base.go

```diff
@@ -148,6 +360,7 @@ func (base *wheeledBase) runAll(ctx context.Context, leftRPM, leftRotations, rig
 	}
 
 	if _, err := rdkutils.RunInParallel(ctx, fs); err != nil {
+		base.logger.Debug("error in run in parallel %#v:", err)
```

Comment:
can we use errors.Wrap rather than putting in debug logging (and mention that we're in `runAll`)?

Reply:
and a similar comment for `setPowerAll`

Reply:
done